test/unit/plugins/welcome.test.js

```diff
@@ -1,8 +1,11 @@
-const assume = require('assume');
-const sinon = require('sinon');
+import assume from 'assume';
+import assumeSinon from 'assume-sinon';
+import sinon from 'sinon';
 
-const WelcomePlugin = require('../../../plugins/welcome');
-const Commenter = require('../../../commenter');
+assume.use(assumeSinon);
+
+import WelcomePlugin from '../../../plugins/welcome/index.js';
+import Commenter from '../../../commenter.js';
 
 const sandbox = sinon.createSandbox();
```

Comment:
Do you still need a sinon sandbox? 99% of the time you just want to use the default sandbox of `sinon` itself